### DISCUSSION TITLE
Issue with station - Azuracast on Linode

### DISCUSSION THREAD

**Discussion by WishBoneBoy007** [AzuraCast/AzuraCast#5663] [General]
Hi team, I have a radio station I want to delete. I get the error "RuntimeException at /var/azuracast/www/src/Entity/Station.php L965: Media storage location not initialized"

I can't access or delete the station from within Azuracast. I'm also looking to add more stations but get the same error. 
I have two radio stations using this database and they work just fine. 
Can someone please help asap? Thank you. 

<img width="1187" alt="Screen Shot 2022-08-10 at 5 34 55 PM" src="https://user-images.githubusercontent.com/88299050/183824245-b1f8603b-8db1-4adf-a6da-160da907cf59.png">

<img width="836" alt="Screen Shot 2022-08-10 at 5 36 27 PM" src="https://user-images.githubusercontent.com/88299050/183824252-8ee00120-4a3a-4847-b356-da9387674e8a.png">

<img width="1034" alt="Screen Shot 2022-08-10 at 5 37 42 PM" src="https://user-images.githubusercontent.com/88299050/183824261-a6249546-be38-41e7-812a-cb1bf4325980.png">
?

**Comment by BusterNeece** [comment]
@WishBoneBoy007 We have resolved this issue completely in the latest Rolling Release version, which will automatically delete stations that were mis-created like this and restore functionality to the main station pages.
